src/org/usfirst/frc/team4571/robot/commands/IntakeCommand.java

```diff
@@ -0,0 +1,50 @@
+package org.usfirst.frc.team4571.robot.commands;
+
+import org.usfirst.frc.team4571.robot.Robot;
+
+import edu.wpi.first.wpilibj.command.Command;
+
+/**
+ *
+ */
+public class IntakeCommand extends Command {
+
+
+	public IntakeCommand() {
+		requires(Robot.INTAKE_SUBSYSTEM);
+		// Use requires() here to declare subsystem dependencies
+		// eg. requires(chassis);
+	}
+
+	// Called just before this Command runs the first time
+	protected void initialize() {
+		Robot.INTAKE_SUBSYSTEM.reverse(0);
```

Comment:
So I noticed that you set "out" as 0 in the subsystem's "initialize" and then set "reverse" to 0 here. This will set both as 0 in the end indeed, but once again, consistency (put them both in one of the initialize methods).